template/src/{% if namespace_package %}{{namespace_package}}{% endif %}/{{ projectname.removeprefix(namespace_package) }}/__init__.py.jinja

```diff
@@ -5,7 +5,7 @@
 import importlib.metadata
 
 try:
-    __version__ = importlib.metadata.version(__package__ or __name__)
+    __version__ = importlib.metadata.version((__package__ or __name__).replace("{{namespace_package}}.", "{{namespace_package}}", 1))
```

Comment:
Given that this now depends on a copier variable, why not this?
```suggestion
    __version__ = importlib.metadata.version("{{projectname}}")
```

Reply:
Didn't know we had that variable, that's better :+1: